components/net/indexeddb/engines/sqlite/create.rs

```diff
@@ -55,5 +55,37 @@ pub(crate) fn create_tables(conn: &rusqlite::Connection) -> Result<(), rusqlite:
             );"#,
         [],
     )?;
+
+    conn.execute(
+        r#"CREATE TABLE index_data
+        (
+            index_id INTEGER NOT NULL,
+            value BLOB NOT NULL,
+            object_data_key BLOB NOT NULL,
+            object_store_id INTEGER NOT NULL,
+            value_locale BLOB,
+            PRIMARY KEY (index_id, value, object_data_key)
+            FOREIGN KEY (index_id) REFERENCES object_store_index(id),
+            FOREIGN KEY (object_store_id, object_data_key)
+            REFERENCES object_data(object_store_id, key)
+        ) WITHOUT ROWID;"#,
```

Comment:
This should be const string literal I think, and it better have documentation to explain what they do. Ditto to other tables in this module.

Reply:
Ah they pretty much are all adapted from firefox: https://github.com/mozilla-firefox/firefox/blob/main/dom/indexedDB/DBSchema.cpp. I'll add a note in the file.